### PR TITLE
Add test_send_basic_output

### PR DESCRIPTION
**Pull request by Thoralf-M**

### CODE REVIEW

File: client/tests/node_api.rs [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // These are E2E test samples, so they are ignored by default.
```
Comment (thibault-martinez) [inline client/tests/node_api.rs:4]:
```suggestion
// These are E2E test samples, so they are ignored by default.

```

File: client/tests/transactions.rs [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// These are E2E test samples, so they are ignored by default.
```
Comment (thibault-martinez) [inline client/tests/transactions.rs:4]:
```suggestion
// These are E2E test samples, so they are ignored by default.

```

File: client/tests/transactions.rs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+mod common;
+
+use common::create_client_and_secret_manager_with_funds;
```
Comment (thibault-martinez) [inline client/tests/transactions.rs]: use self::common and put it after

File: client/tests/common/mod.rs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                QueryParameter::HasTimelock(false),
+                QueryParameter::HasStorageDepositReturn(false),
+            ])
+            .await?;
+
+        if !output_ids.is_empty() {
+            return Ok((client, secret_manager));
+        }
+    }
+    panic!("Faucet no longer wants to hand over coins");
```
Comment (thibault-martinez) [inline client/tests/common/mod.rs:53]: 😄

File: client/tests/transactions.rs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        payload::transaction::TransactionEssence,
+    },
+    node_api::indexer::query_parameters::QueryParameter,
+    Result,
+};
+use iota_types::block::{output::OutputId, payload::Payload};
+
+#[ignore]
+#[tokio::test]
+async fn test_send_basic_output() -> Result<()> {
```
Comment (thibault-martinez) [inline client/tests/transactions.rs]:
```suggestion
async fn send_basic_output() -> Result<()> {
```

**Review by thibault-martinez — DISMISSED** [review]

File: client/tests/common/mod.rs [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+pub use self::constants::{FAUCET_URL, NODE_LOCAL};
+
+/// Sets up a Client with node health ignored.
+pub fn setup_client_with_node_health_ignored() -> Client {
+    Client::builder()
+        .with_node(NODE_LOCAL)
+        .unwrap()
+        .with_ignore_node_health()
```
Comment (kwek20) [inline client/tests/common/mod.rs]: do we really need ignore_nodehealth? its not in the wallet tests

Comment (Thoralf-M) [reply client/tests/common/mod.rs]: No, would just not return an error even if the node is not healthy

**Review by kwek20 — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**